README.rst

```diff
@@ -51,5 +51,5 @@ Supported versions
 **django-treebeard** officially supports
 
 * Django 1.8 - 1.10
-* Python 2.7, 3.4, 3.5
+* Python 2.7, 3.4, 3.5, 3.6
```

Comment:
If you add Python 3.6 to the testing matrix, you should also add it to the ``classifiers`` in ``setup.py``.